mllib/src/test/scala/org/apache/spark/ml/regression/DecisionTreeRegressorSuite.scala

```diff
@@ -96,6 +108,22 @@ class DecisionTreeRegressorSuite
       assert(variance === expectedVariance,
         s"Expected variance $expectedVariance but got $variance.")
     }
+
+    val toyDF = TreeTests.setMetadata(toyData, Map.empty[Int, Int], 0)
+    dt.setMaxDepth(1)
+      .setMaxBins(6)
+      .setSeed(0)
+    val calculatedVariances = dt.fit(toyDF).transform(toyDF).select("variance").collect().map {
```

Comment:
Can we use `.select(dt.getVarianceCol)` instead? Also, the closing brace should go on the next line, like:

```scala
.map {
  case Row(...)
}
```